rust-runtime/aws-smithy-http-server/examples/pokemon-service/src/lib.rs

```diff
@@ -0,0 +1,232 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+//! Pokémon Service
+//!
+//! This crate implements the Pokémon Service.
+#![warn(missing_docs, missing_debug_implementations, rust_2018_idioms)]
+use std::{
+    collections::HashMap,
+    convert::TryInto,
+    sync::{atomic::AtomicU64, Arc},
+};
+
+use aws_smithy_http_server::Extension;
+use pokemon_service_sdk::{error, input, model, output};
+use tracing::{debug, error};
```

Comment:
I'm a fan of not importing `tracing` or `log`, it makes logging statements more apparent in the code.

Reply:
I don't have an opinion on this, I can change it to use the full namespace and remove the use.